jest-puppeteer.config.js

```diff
@@ -7,8 +18,8 @@ module.exports = {
     ],
   },
   server: {
+    command: `yarn run server ${SERVER_PORT}`, // This takes longer & times out: `PORT=${SERVER_PORT} BROWSER=none yarn start`
```

Comment:
starting the dev server does take longer, but i feel like its worth it to not have to worry about building before testing. the suite passes for me and jenkins and travis, what issue were you having?

Reply:
Yep, I would also rather use the dev server at this point... when I use the dev server though, The 2 browser tests fail, not from timeouts but from being blank (could still be because of a HTTP request for the JS file though):

- `Browser sanity tests › displays the "Hello developer" console text`
- `Browser sanity tests › loads the 404 page`

Maybe I can add a line after `await page.goto(...)` that checks to make sure the JS file loaded

Reply:
oh, Jenkins fails in the same way as my local machine: https://jenkins-dev.cnx.org:8080/blue/organizations/jenkins/openstax-continuous-integration%2Frex-web/detail/PR-18/1/pipeline/#step-20-log-155